docs/apidiffs/current_vs_latest/opentelemetry-spring-boot.txt

```diff
@@ -0,0 +1,137 @@
+Comparing source compatibility of opentelemetry-spring-boot-2.6.0-SNAPSHOT.jar against 
++++  NEW ANNOTATION: PUBLIC(+) ABSTRACT(+) io.opentelemetry.instrumentation.spring.autoconfigure.EnableOpenTelemetry  (not serializable)
```

Comment:
I couldn't find any usage of this annotation, is there some magic involved? thanks

Reply:
yes, `@ComponentScan(basePackages = "io.opentelemetry.instrumentation.spring.autoconfigure")` is the magic

@jeanbisutti I think we didn't document this anywhere